### ISSUE TITLE
Tutorial Aggregating CSV Data (Spatial Binning) Issue

### ISSUE THREAD

**Issue by dataunleashed** [Esri/gis-tools-for-hadoop#30]
I am using Cloudera CDH 5.3.3 with Hive 0.13.1 (/opt/cloudera/parcels/CDH-5.3.3-1.cdh5.3.3.p0.5/jars/hive-common-0.13.1-cdh5.3.3.jar) on a 4-node cluster VM environment. 
I followed the Aggregating CSV Data tutorial (https://github.com/Esri/gis-tools-for-hadoop/wiki/Aggregating-CSV-Data-%28Spatial-Binning%29) and everything appeared to be working. However, after step 11, I ran a simple SELECT query and it returned nothing even though the files existed in the Hive folder. As a result, when I followed step 12 to get the result taxi_agg table to ArcMap, the json file is empty with no data.

Here are some screenshots:
1. Table taxi_demo location:
   ![image](https://cloud.githubusercontent.com/assets/4471844/7705181/50fffb30-fe08-11e4-841f-84e88fc318e2.png)
2. Files in the HDFS folder for Hive taxi_demo table:
   ![image](https://cloud.githubusercontent.com/assets/4471844/7705238/b1c04d9e-fe08-11e4-9457-e041e8cc522c.png)
3. Data in the taxi_demo table Hive folder:
   ![image](https://cloud.githubusercontent.com/assets/4471844/7705279/e7abe3dc-fe08-11e4-96e6-225577071399.png)
4. Hive query returns nothing:
   ![image](https://cloud.githubusercontent.com/assets/4471844/7705342/44d62482-fe09-11e4-90c7-2b01ff60c3d1.png)

![image](https://cloud.githubusercontent.com/assets/4471844/7705381/799d979a-fe09-11e4-82df-22dc195ccc83.png)

Please help.

Thanks,

Tom

**Comment by smambrose** [comment]
Can you try the solution at the bottom here : https://github.com/Esri/gis-tools-for-hadoop/issues/25

Please let us know if it works, or if you run into further issues.

**Comment by dataunleashed** [comment]
@smambrose, thank you for your quick response. I've followed the solution at the bottom of #25 as you referenced and it didn't work either. Very much everything worked and I could see the JSON-format result bins in the output file in the Hive folder, but when the I ran the Hive query or the ArcMap Copy from HDFS tool, I just got an empty file.

`hive> select * from agg_samp limit 1;`
`OK`
`Time taken: 0.048 seconds`

I don't know if this is a Cloudera CDH 5.3.3 issue or not, but it worked fine when I tried the solution on the latest Hortonworks HDP 2.2.4 Sandbox with Hive 0.14. Is there anything else that I can try or may have overlooked? 

We are looking to implement your great work on the ESRI for Hadoop Framework on my company Cloudera production cluster, any help would be greatly appreciated.

Thanks,

Tom
